internal/util/projutil/project_util.go

```diff
@@ -208,6 +216,12 @@ func IsOperatorGo() bool {
 	return err == nil || os.IsExist(err)
 }
 
+// todo(camilamacedo86): it should be removed when we no longer support the old scaffold layout
+// IsNewOperatorLayout return true when has the project file
+func IsNewOperatorLayout() bool {
+	return kbutil.HasProjectFile()
+}
```

Comment:
This seems duplicative and unnecessary. Just use `kbutil.HasProjectFile()` wherever this would be used.